### ISSUE TITLE
Cannot Update Company Information ir_ui_view.py Field `social_twitter` does not exist

### ISSUE THREAD

**Issue by Jirawatz** [odoo/odoo#60673]
Impacted versions:13

Steps to reproduce: Installed Odoo 13.0-20201024 through https://nightly.odoo.com/13.0/nightly/ on Ubuntu 20.04

Current behavior: Error when update company information
```
Error:
Odoo Server Error

Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 624, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 310, in _handle_exception
    raise pycompat.reraise(type(exception), exception, sys.exc_info()[2])
  File "/usr/lib/python3/dist-packages/odoo/tools/pycompat.py", line 14, in reraise
    raise value
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 669, in dispatch
    result = self._call_function(**self.params)
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 350, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/usr/lib/python3/dist-packages/odoo/service/model.py", line 94, in wrapper
    return f(dbname, *args, **kwargs)
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 339, in checked_call
    result = self.endpoint(*a, **kw)
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 915, in __call__
    return self.method(*args, **kw)
  File "/usr/lib/python3/dist-packages/odoo/http.py", line 515, in response_wrap
    response = f(*args, **kw)
  File "/usr/lib/python3/dist-packages/odoo/addons/web/controllers/main.py", line 1322, in call_kw
    return self._call_kw(model, method, args, kwargs)
  File "/usr/lib/python3/dist-packages/odoo/addons/web/controllers/main.py", line 1314, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/usr/lib/python3/dist-packages/odoo/api.py", line 383, in call_kw
    result = _call_kw_model(method, model, args, kwargs)
  File "/usr/lib/python3/dist-packages/odoo/api.py", line 356, in _call_kw_model
    result = method(recs, *args, **kwargs)
  File "/usr/lib/python3/dist-packages/odoo/models.py", line 1463, in load_views
    result['fields_views'] = {
  File "/usr/lib/python3/dist-packages/odoo/models.py", line 1464, in <dictcomp>
    v_type: self.fields_view_get(v_id, v_type if v_type != 'list' else 'tree',
  File "/usr/lib/python3/dist-packages/odoo/addons/web/models/models.py", line 206, in fields_view_get
    r = super().fields_view_get(view_id, view_type, toolbar, submenu)
  File "/usr/lib/python3/dist-packages/odoo/models.py", line 1553, in fields_view_get
    xarch, xfields = View.postprocess_and_fields(self._name, etree.fromstring(result['arch']), view_id)
  File "/usr/lib/python3/dist-packages/odoo/addons/base/models/ir_ui_view.py", line 987, in postprocess_and_fields
    self.raise_view_error(message, view_id)
  File "/usr/lib/python3/dist-packages/odoo/addons/base/models/ir_ui_view.py", line 614, in raise_view_error
    raise ValueError(message)
ValueError: Field `social_twitter` does not exist

Error context:
View `n/a`
[view_id: n/a, xml_id: n/a, model: n/a, parent_id: n/a]
```
Expected behavior: Update Company page

Video/Screenshot link (optional):

**Comment by pedrobaeza** [comment]
Hi, thanks for reporting. Are you able to reproduce the problem in runbot infrastructure (http://runbot.odoo.com/runbot - click on the blue door icon of the version row - admin/admin)?

If so, please explain with all the possible detailed steps to get to the error. Not having them, it's very difficult to hunt down the problem. Please don't be shy of adding details, and even better if you provide some screenshots or videos.

If not, this can be due to a configuration change, a custom/community module you have deployed in your DB or an outdated code. Please report back if it's the case.

For keeping a clean house in this issue tracker, if you don't answer in several days, this will be closed assuming it was one of the stated things.

**Comment by Jirawatz** [comment]
I cannot reproduce the error on the runbot infrastructure. However, I have tried the following:
1. I have tried to uninstall all of the modules that I have deployed (Will that be sufficient enough to pinpoint whether the error came from the odoo or from a third-party module?) If it is sufficient, then I have uninstalled all of the third-party modules from my DB the issue is still not resolved.
2. I have updated my odoo to the latest version 13-20201025

Here is a snapshot of the issue
![Screen Recording 2563-10-26 at 16 42 39](https://user-images.githubusercontent.com/31199694/97159344-4f2c0c80-17ad-11eb-8083-e411f16852db.gif)

Is there any way I can do to help me pinpoint the issue? 
(Sorry I am still a newbie)
